contextx/runner.go

```diff
@@ -2,39 +2,33 @@ package contextx
 
 import "context"
 
-// Runner is the basic behaviour of a Server, which will run some logic. It
-// receives a Context to allow the implementation of several patterns like
-// deadlines and cancellation signals, parameter bags, dependency injection
-// container, etc.
+// A Runner allows to run a context-aware process to get advantage of
+// context deadlines, cancelation signals, and other request-scoped values
+// across API boundaries and between such processes.
 type Runner interface {
 	Run(context.Context)
 }
 
-// RunnerFunc is the functional countertype of an Runner, it allows the use of
-// ordinary functions as a Runner.
+// The RunnerFunc type is an adapter to allow the use of ordinary functions as Runners.
 type RunnerFunc func(context.Context)
 
-// Run bust be implemented by RunnerFunc to implements Runner, so when someone
-// calls the Run function over an element of this type it will execute itself.
+// Run calls the wrapped function with the provided context.
 func (f RunnerFunc) Run(ctx context.Context) {
 	f(ctx)
 }
 
-// EmptyRunner is a Runner that does nothing for testing purposes only.
-func EmptyRunner() RunnerFunc {
-	return func(context.Context) {
-		// don't do anything
-	}
+// NopRunner returns a no-op Runner that does nothing.
+func NopRunner() RunnerFunc {
```

Comment:
When thinking of the testing scenario, I don't understand how this is different from EmptyRunner, which has been deleted in the same PR.

I'm not convinced we should provide a NopRunner, but I guess it can have it's uses.